tools/download_from_gcs.py

```diff
@@ -102,9 +95,12 @@ def MaybeDownloadFileFromGcs(bucket, sha1_file, output_file, force=False):
                       output_file)
         return False
 
-  tmp_file = _DownloadFromGcsAndCheckSha1(bucket, sha1)
-  if not tmp_file:
-    return False
+  try:
+    tmp_file = _DownloadFromGcsAndCheckSha1(bucket, sha1)
+  except (urllib.error.URLError, GcsDownloadException) as e:
+    logging.error('Download failed: \'%s\', retrying', str(e))
+    time.sleep(1)
```

Comment:
Are we able to reuse the retry decorator we use in raspi?

Reply:
I did in https://github.com/youtube/cobalt/pull/1463 but @kaidokert thinks it's a bad idea. For a simple retry here he thought something simple like this would be better.

I don't have a strong preference either way.

Reply:
I should add that that PR is not entirely complete. It needs the new exception to cover the sha1 mismatch case.

Reply:
I'll revisit this if this PR doesn't fix the issues.